README.md

```diff
@@ -53,22 +59,21 @@ sample_tests.zip
 ```
 
 ## Description of test files
-The tests to be run on the students' submissions are in `autograde.py`. `prepend.py` consists of code to be prepended to the submissions before the tests are run, while `append.py` consists of code to be appended to the submissions before the tests are run.
 
-The tests for a student's programming exercise are run by sending the package with the templates in the `submission` folder replaced by the student's submissions to the code evaluator. The evaluator generates a xml test report file by running `make test` on the package. The test report file is then retrieved by Coursemology to render the test results for the student.
+* The tests to be run on the students' submissions are in `autograde.py`. `prepend.py` consists of code to be prepended to the submissions before the tests are run, while `append.py` consists of code to be appended to the submissions before the tests are run.
+* The tests for a student's programming exercise are run by sending the package with the templates in the `submission` folder replaced by the student's submissions to the code evaluator. The evaluator generates a xml test report file by running `make test` on the package. The test report file is then retrieved by Coursemology to render the test results for the student.
 
 ### autograde.py
-Each test to be executed on the students' code submissions must be written as methods of a class extending `unittest.TestCase` as required by the unit test framework.
+* Each test to be executed on the students' code submissions must be written as methods of a class extending `unittest.TestCase` as required by the unit test framework.
 
-The class must have the `setUp` instance method defined in the following manner:
+* The class must have the `setUp` instance method defined in the following manner:
 ```
 def setUp(self):
         self.meta = { 'expression': '', 'expected': '', 'hint': '' }
 ```
 Public test cases are instance methods with names of the format: `test_public_*`
```

Comment:
It's a bit strange that public tests and private tests are not separate bullet points while they 2 items below are. Why did you arrange it this way?